release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -214,6 +206,13 @@ private IEnumerator InitializeOnNextFrame()
 		objUnityInput.placeholder.gameObject.SetActive(false);
 		objUnityText.enabled = false;
 		objUnityInput.enabled = false;
+
+		yield return null;
```

Comment:
This would benefit from a comment. Maybe something like: "For iOS autofill to work correctly, all input fields expected to be autofilled, need to exist before we set the native focus. Assuming all related input fields are created simultaneously, they will all be initialized after one frame (see above). In this case, waiting for one more frame should ensure they all exist before we set the focus. In more complex cases, the caller may need to further delay setting focus, until all related input fields are guaranteed to exist."

Maybe that's too extensive, though. Feel free to shorten that, if you don't think all of it adds value.

Reply:
Love it when added code gets more lines of comments to explain them than there originally were :D
Fixed.